blocks/cards/cards.js

```diff
@@ -10,7 +10,29 @@ export default function decorate(block) {
     while (row.firstElementChild) li.append(row.firstElementChild);
     [...li.children].forEach((div) => {
       if (div.children.length === 1 && div.querySelector('picture')) div.className = 'cards-card-image';
-      else div.className = 'cards-card-body';
+      else { div.className = 'cards-card-body'; }
+
+      /* cards simple */
+      if (block.classList.contains('cards') && block.classList.contains('simple')) {
```

Comment:
`block.classList.contains('cards')` will always be true within this code

Reply:
Updated.